src/locale/translations/en.json

```diff
@@ -49,10 +49,19 @@
       },
       "Body2": "You do not have to agree, even if you agreed the day before.",
       "Body3": "If you’re worried about your symptoms, get medical care as soon as you can. Take care of yourself and your family during this time.",
-      "TipTitle": "TIP: ",
-      "TipBody": "You can prepare for your call from Public Health through the test results website.  ",
-      "TipLinkText": "Test results website",
-      "TipURL": "https://covid19results.ehealthontario.ca:4443/login"
+      "Tip": {
+        "Title": "TIP: ",
+        "ON": {
+          "Body": "You can prepare for your call from Public Health through the test results website.",
+          "CTA": "Test results website",
+          "URL": "https://covid19results.ehealthontario.ca:4443/login"
+        },
+        "NL": {
+          "Body": "You need to self-isolate and follow advice from public health.",
+          "CTA": "How to self-isolate",
+          "URL": "https://www.gov.nl.ca/covid-19/files/factsheet-covid-19-how-to-self-isolate-1.pdf"
```

Comment:
This URL is for a pdf so not ideal - @katewilhelm is looking into what to do here.

Reply:
On my phone - the browser ends up asking to download the PDf and I end up on an empty browser page.

Reply:
That is not a great experience. Let's revisit this in a future PR.